README.md

```diff
@@ -81,6 +82,13 @@ If you want to save the results of a custom SQL query, do this:
 
 The `--output` option specifies the table that should contain the results of the query.
 
+## Using db-to-sqlite with Postgres schemas
+If the tables you want to copy from your PostgreSQL database aren't in the default schema, you can specify an alternate one with the `--postgres_schema` option:
+
+    db-to-sqlite "postgresql://localhost/myblog" blog.db \
+        --all \
+        --postgres_schema my_schema
```

Comment:
One tiny change: this should be `--postgres-schema` for consistency with the `--index-fks` and `--no-index-fks` options.

Reply:
Gotcha, updated!